An Untitled Mining Game/An Untitled Mining Game/objects/obj_item/Step_0.gml

```diff
@@ -0,0 +1,64 @@
+if (drop_move) {
+	x = lerp(x, x_goal, .1);	
+	y = lerp(y, y_goal, .1);
```

Comment:
I'm glad that the mention of linear interpolation has been useful! It's used here to great effect.